markpar/WordPress-iOS
# Issue/10623 add dependency attribution

Fixes #10623.

This fix implements the great suggestion from @jtreanor to convert the 3rd-party acknowledgements markdown created for us by CocoaPods into HTML and then display that in a web view.  Using the `commonmarker` gem, we convert the markdown to HTML during the `post_install` phase of `pod install`, adding some lightweight global styling along the way.  The resulting HTML is embedded as a resource in the main app bundle as `acknowledgements.html` and this is then displayed in a web view when the user taps `Acknowledgements` on the `About WordPress for iOS` screen.

Unit tests in Xcode (Product > Test) passed. 

To test:
- Boot app.
- Me > App Settings > About WordPress for iOS > Acknowledgements.

Expect:  A list of licenses for third-party libraries we use.  Content should be identical to `Pods/Target Support Files/Pods-WordPress/Pods-WordPress-acknowledgements.markdown`

Update release notes:
